src/SelfServe/Decorators.tsx

```diff
@@ -130,12 +191,18 @@ export const Values = (inputOptions: InputOptions): PropertyDecorator => {
   }
 };
 
+/**
+ * Indicates to the compiler that UI should be generated from this class.
+ */
 export const IsDisplayable = (): ClassDecorator => {
   return (target) => {
     buildSmartUiDescriptor(target.name, target.prototype);
   };
 };
 
+/**
+ * Indicates how often the auto refresh of the component should take place
```

Comment:
>component [](http://example.com/codeflow?start=47&length=9)

auto refresh of the page.. dont use the word component.

also, please add a one liner before this: if there is long running operation in your page after submit happens, the page can optionally auto refresh itself.. .....